packages/lpa-questionnaire-web-app/src/validators/extra-conditions.js

```diff
@@ -0,0 +1,23 @@
+const { body } = require('express-validator');
+
+const rules = () => {
+  const extraConditionsRef = 'extra-conditions';
```

Comment:
Looks like this needs to be rebased with the feature file. This shouldn't be in this PR